sklearn/feature_extraction/text.py

```diff
@@ -390,28 +390,6 @@ def _check_stop_words_consistency(self, stop_words, preprocess, tokenize):
             self._stop_words_id = id(self.stop_words)
             return 'error'
 
-    def _validate_custom_analyzer(self):
```

Comment:
Seems like a github glitch for the diff since I remember merging this in an earlier PR and this function is not in master.